src/runtime/lpython_builtin.py

```diff
@@ -129,7 +129,7 @@ def pow(x: f64, y: i32) -> f64:
 def pow(x: bool, y: bool) -> i32:
     if y and not x:
         return 0
-
+    
```

Comment:
Let's remove these spaces and instead fix our tokenizer/parser (https://github.com/lcompilers/lpython/issues/2725).